src/ng/animateRunner.js

```diff
@@ -85,7 +85,7 @@ var $$AnimateRunnerFactoryProvider = function() {
 
         // the document may not be ready or attached
         // to the module for some internal tests
-        if (doc && doc.hidden) {
+        if ($$isDocumentHidden()) {
```

Comment:
Instead of doing this in every tick (and for every runner), coudn't we listen for an event and update the function as necessary ?